src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -666,6 +666,17 @@
      */
     boolean containsInventory(Inventory inventory);
 
+    /**
+     * Transforms this inventory using the transformation
```

Comment:
I would state that its transformed using the **given** transformation. Also add a dot at the end of the line.